### PR TITLE
Ember upgrade v4.12 LTS

### PR DESCRIPTION
**Pull request by benjay10**
This is a large PR again, but difficult to split up now.

What has been done here:

* Ember upgrade to v4.12 LTS
* Reviewed all templates to use newer AppUniversum components and style
* Cleaned up packages
* Reviewed Docker build system
* Resolved deprecations
* Made sure that it works with the frontend behind the dispatcher

### CODE REVIEW

File: .github/workflows/ci.yml
[the remark this answers is not in the document]
Comment (benjay10) [reply .github/workflows/ci.yml:1]: Ah good catch! Removed it.

**Review by Windvis — COMMENTED** [review]
Code changes look good to me 👍 Some small nitpicks.

File: app/styles/app.scss [hunk 1]
```diff
@@ -1,111 +1,17 @@
...
-@import "appuniversum/u-spacings";
-@import "appuniversum/u-responsive-spacings";
-@import "appuniversum/u-print";
-@import "appuniversum/u-hide";
-@import "appuniversum/u-visible";
-
-
-// TEMPORARY HACKS AND QUICKFIXES
-@import 'shame';
+@import "_shame";
```
[the remark this answers is not in the document]
Comment (benjay10) [reply app/styles/app.scss:17]: Does that mean that the file itself should also be without underscore?

[the remark this answers is not in the document]
Comment (Windvis) [reply app/styles/app.scss:17]: No, [partials](https://sass-lang.com/guide/#partials) have a leading underscore 👍

**Review by Windvis — APPROVED** [review]

**Pull request merged**